wizeline/wize-q-front
# Feat/display answers

WIP-215 - Tabs layout for the open questions and the list of the answered ones.
#### What does this PR do?
Integrates a tab component and a list of answered questions
#### Where should the reviewer start?
Install the new scripts included in the package.json file
#### How should this be manually tested?
once the new packages are installed, just run the project and enter to [http://localhost:3000/all_questions](http://localhost:3000/all_questions) there should appear two tabs. Clicking in each tab should change between the table of questions and the answered ones
#### Any background context you want to provide?
QuestionsAnswered.js was builded with List.js as base.
#### What are the relevant tickets?
[https://wizeline.atlassian.net/browse/WIP-215](WIP-215)
#### Screenshots
![screencapture-localhost-3000-all_questions-1495143135656](https://cloud.githubusercontent.com/assets/28713208/26224355/9572d12e-3be7-11e7-8d92-cb277854daf6.png)
![screencapture-localhost-3000-all_questions-1495143158739](https://cloud.githubusercontent.com/assets/28713208/26224364/a0030a96-3be7-11e7-8558-406bbdfe3c00.png)
